src/style.css

```diff
@@ -0,0 +1,71 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
```

Comment:
[checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) point 7, you should connect two `font-weight`s, this will fix tests

Reply:
I changed font face, but it didn't resolve the issue with font, tests failed :(